src/test/java/test/dependent/github1380/GitHub1380Sample4.java

```diff
@@ -0,0 +1,21 @@
+package test.dependent.github1380;
```

Comment:
Oh, ok. I don't know if someone would to do it a day. But it was easier sample (and not a unit test of graph) which reproduces the issue.